examples/filters.rs

```diff
@@ -112,9 +118,9 @@ impl RequestLogger {
 
 impl ContextFilter for RequestLogger {
     ///Count requests and log the path.
-    fn modify(&self, ctx: FilterContext, context: &mut Context) -> ContextAction {
+    fn modify(&self, _ctx: FilterContext, context: &mut Context) -> ContextAction {
         *self.counter.write().unwrap() += 1;
-        ctx.log.note(&format!("Request #{} is to '{}'", *self.counter.read().unwrap(), context.uri));
+        info!("Request #{} is to '{}'", *self.counter.read().unwrap(), context.uri);
```

Comment:
Yeah I think that is good idea. We can actually conditionally print. If program is run with a log level that will not print this line, we can print instructions to run with log level debug to see some information.